lib/PuppeteerSharp/FrameManager.cs

```diff
@@ -414,12 +414,20 @@ private async Task EnsureSecondaryDOMWorldAsync(string name)
                 Source = $"//# sourceURL={ExecutionContext.EvaluationScriptUrl}",
                 WorldName = name,
             });
-            await Task.WhenAll(GetFrames().Select(frame => Client.SendAsync("Page.createIsolatedWorld", new PageCreateIsolatedWorldRequest
+
+            try
+            {
+                await Task.WhenAll(GetFrames().Select(frame => Client.SendAsync("Page.createIsolatedWorld", new PageCreateIsolatedWorldRequest
```

Comment:
Have you established a max line length? Just asking.

Reply:
My desired file width is set to 120. We are not enforcing that be we could :)